hono/test/utils/factory/items.ts

```diff
@@ -1,64 +1,129 @@
-export function validItem() {
-  return {
-    name: 'Vintage Denim Pants',
-    brand: 'Levi',
-    photoUrl: 'https://example.com/',
-    type: 'top',
-    rating: 2,
+import { faker } from '@faker-js/faker'
+import { drizzle } from 'drizzle-orm/postgres-js'
+import postgres from 'postgres'
+import * as schema from '../../../src/schema'
+import { items, itemTypeEnum } from '../../../src/schema'
+
+export type ItemType = 'layer' | 'top' | 'bottom' | 'footwear' | 'accessory'
+
+export interface Item {
```

Comment:
Two very similar interfaces: `Item` and `ItemAPI`. Not sure if there is a better way to approach this. Something we can also try doing is putting all the interfaces in file then import them to have cleaner code.

Reply:
I concur. I left this in since there is a good chance ItemAPI will change once the materialized view is implemented.

Reply:
Good point